src/components/Button.js

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+function Button({ name }) {
+  return (
+    <div>
+      {name}
+    </div>
+  );
+}
+
+Button.propTypes = {
+  name: PropTypes.string,
+};
+
+Button.defaultProps = {
+  name: 'red',
+};
```

Comment:
`Button` component dont need a defaultValue, all their values will be loaded in the `ButtonPanel` component, you may delete this